src/appleseed/renderer/modeling/frame/frame.cpp

```diff
@@ -172,6 +177,23 @@ Frame::Frame(
     // Retrieve the image properties.
     m_props = impl->m_image->properties();
 
+    // Load the reference image.
+    if (!impl->m_ref_image_path.empty())
+    {
+        RENDERER_LOG_DEBUG("loading reference image %s...", impl->m_ref_image_path.c_str());
+
+        GenericImageFileReader reader;
+        impl->m_ref_image.reset(reader.read(
+            search_paths.qualify(impl->m_ref_image_path).c_str()));
+
+        if (!has_valid_ref_image())
+        {
+            RENDERER_LOG_ERROR(
+                "the reference image is not compatible with the output frame "
+                "(different dimensions, tile size or number of channels).");
```

Comment:
The "tile size" part is no longer true, AFAICT.

Reply:
Yep, channel size neither.